src/PhpSpreadsheet/Reader/Xlsx.php

```diff
@@ -546,6 +546,13 @@ public function load($pFilename)
                         'SimpleXMLElement',
                         Settings::getLibXmlLoaderOptions()
                     );
+                    if ($xmlStyles->colors->indexedColors) {
+                        $colors = [];
+                        foreach ($xmlStyles->colors->indexedColors as $rgbColor) {
+                            $colors[] = (string) $rgbColor['rgb'];
```

Comment:
The index here starts at 0, but our hardcoded `$indexedColors` starts at 1. Is this intended ?

Reply:
Did you have time to double-check that index thing ?

Reply:
Unfortunately, I've not had chance to check this out yet. Please keep it open for now.